core/src/main/java/tc/oc/pgm/variables/types/TimeLimitVariable.java

```diff
@@ -0,0 +1,77 @@
+package tc.oc.pgm.variables.types;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.party.VictoryCondition;
+import tc.oc.pgm.timelimit.TimeLimit;
+import tc.oc.pgm.timelimit.TimeLimitMatchModule;
+import tc.oc.pgm.variables.VariableDefinition;
+
+public class TimeLimitVariable extends AbstractVariable<Match> {
+
+  VictoryCondition result = null;
+  Duration overtime = null;
+  Duration maxOvertime = null;
+  Duration endOvertime = null;
+  boolean show = true;
```

Comment:
could probs just be a TimeLimit object, also private?
it should be fine as long as TL is just a plain immutable object with no extra logic, which i believe it is

Reply:
Oh yeah that would be better